src/base/torrentfilter.h

```diff
@@ -60,6 +60,7 @@ class TorrentFilter
 
     static const QString AnyCategory;
     static const QStringSet AnyHash;
+    static const QString TagFilteringDisabled;
```

Comment:
Yeah, I went back and forth on whether to call it AnyTag. This issue is that what these things functionally mean is "select all torrents regardless of hash / category / tag".

AnyHash is an accurate name because all torrents have a hash.
AnyTag is not accurate because not all torrents have tags. Thus this filter would also include torrents that do not have "any tag".

Perhaps it would be better if they were all changed to *FilteringDisabled to be consistent.